app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -278,6 +284,9 @@ input.top-menu-search--input
       right: 15px
       top: calc(50% - 10px / 2)
 
+  .menu-drop-down-container
+    overflow: hidden !important
```

Comment:
Is the `!important` necessary? Can you please check whether there is way to avoid this?

Reply:
Done! Thanks 👍